docs/source/openapi.json

```diff
@@ -4620,12 +4623,13 @@
                 "examples": {
                   "all the capabilities": {
                     "summary": "valid dataset",
-                    "description": "Try with https://datasets-server.huggingface.co/is-valid?dataset=glue",
+                    "description": "Try with https://datasets-server.huggingface.co/is-valid?dataset=nyu-mll/glue",
```

Comment:
glue is now a community dataset :tada: 
note that we will have to go through all our docs when all canonical datasets will be transferred and update them

Reply:
oh right! It's broken on our side: the Hub redirects to the new one, but we don't have the information on the datasets server.

I opened https://github.com/huggingface/datasets-server/issues/2578